spec/javascript/e2e/SupplementalApplicationPage/preferences.e2e.js

```diff
@@ -0,0 +1,61 @@
+import puppeteer from 'puppeteer'
```

Comment:
small nit: filenames aren't super consistent for these new broken-out things.

action_buttons
preferences
manage_rental_assistance
status_history

I think it might be a bit clearer if we pick a convention and stick with it. 

I'd prefer if we keep them to domain-related topics, e.g. 
-preferences
-rental_assistance
-status_update (combine status history and action_buttons)

Reply:
sounds good to me, done!